core/modules/update/theme/update.theme.inc

```diff
@@ -105,7 +108,12 @@ function theme_update_report($variables) {
 
     $row = '<div class="version-status">';
     $status_label = theme('update_status_label', array('status' => $project['status']));
-    $row .= !empty($status_label) ? $status_label : check_plain($project['reason']);
+    if (empty($status_label)) $status_label = check_plain($project['reason']);
+    if ($updatelink) {
+      $row .= l($status_label,'admin/'.$project['project_type'].'s/update',array('html' => TRUE));
```

Comment:
Thanks -- modified those now and will keep in mind going forward! Thanks for the link to the documentation, I hadn't bookmarked that before.